src/chrome/content/rules/WolverineWorldwide.com.xml

```diff
@@ -1,6 +1,15 @@
+<!--
+	Timed out:
+		- img.wolverineworld.com, equivalent to wolverine-o.scene7.com
+-->
+
 <ruleset name="WolverineWorldwide.com">
 	<target host="wolverineworldwide.com" />
 	<target host="www.wolverineworldwide.com" />
+	<target host="img.wolverineworldwide.com" />
+
+	<rule from="^http://img\.wolverineworldwide\.com/"
+		to="https://wolverine-o.scene7.com/" />
```

Comment:
Please add a test url for redirects to other hosts.

Reply:
Added, thanks.